exercises/03-sagas/README.md

```diff
@@ -14,6 +14,10 @@ The following events are currently published in the system:
 
 NOTE: In order to view saga details in ServiceInsight, the [SagaAudit Plugin](https://docs.particular.net/nservicebus/sagas/saga-audit) must be installed in the endpoints which contain the sagas. Currently, the solution does not have this plugin installed so you will have to install it yourself before running the endpoints, if you wish to view the details of your sagas in ServiceInsight.
```

Comment:
The URL should be changed to https://docs.particular.net/nservicebus/sagas/saga-audit?version=sagaaudit_2